addOns/paramminer/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+Rename multiple options in GUI and documentation.
```

Comment:
Once rebased this should be under a `### Changed` section, should also start with a hyphen.